src/protocol/attribution/accumulate_credit.rs

```diff
@@ -411,14 +371,13 @@ mod tests {
     }
 
     #[tokio::test]
-    pub async fn test_reshare() {
+    pub async fn reshare() {
```

Comment:
I kept it because it appears to be testing that the structure here is resharable.  Though I guess the fact that the code that uses this compiles is a better test :)